Example/Example/UIColor.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  UIColor.swift
+//  Example
+//
+//  Created by Pedro Vicente Gomez on 18/12/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIColor {
+
+    convenience init(rgba: String) {
+        var red:   CGFloat = 0.0
+        var green: CGFloat = 0.0
+        var blue:  CGFloat = 0.0
+        var alpha: CGFloat = 1.0
+
+        if rgba.hasPrefix("#") {
+            let index   = rgba.startIndex.advancedBy(1)
+            let hex     = rgba.substringFromIndex(index)
+            let scanner = NSScanner(string: hex)
+            var hexValue: CUnsignedLongLong = 0
+            if scanner.scanHexLongLong(&hexValue) {
+                switch (hex.characters.count) {
+                case 3:
+                    red   = CGFloat((hexValue & 0xF00) >> 8)       / 15.0
+                    green = CGFloat((hexValue & 0x0F0) >> 4)       / 15.0
+                    blue  = CGFloat(hexValue & 0x00F)              / 15.0
+                case 4:
+                    red   = CGFloat((hexValue & 0xF000) >> 12)     / 15.0
+                    green = CGFloat((hexValue & 0x0F00) >> 8)      / 15.0
+                    blue  = CGFloat((hexValue & 0x00F0) >> 4)      / 15.0
+                    alpha = CGFloat(hexValue & 0x000F)             / 15.0
+                case 6:
+                    red   = CGFloat((hexValue & 0xFF0000) >> 16)   / 255.0
+                    green = CGFloat((hexValue & 0x00FF00) >> 8)    / 255.0
+                    blue  = CGFloat(hexValue & 0x0000FF)           / 255.0
+                case 8:
+                    red   = CGFloat((hexValue & 0xFF000000) >> 24) / 255.0
+                    green = CGFloat((hexValue & 0x00FF0000) >> 16) / 255.0
+                    blue  = CGFloat((hexValue & 0x0000FF00) >> 8)  / 255.0
+                    alpha = CGFloat(hexValue & 0x000000FF)         / 255.0
+                default:
+                    print("Invalid RGB string, number of characters after '#' should be either 3, 4, 6 or 8", terminator: "")
+                }
+            } else {
+                print("Scan hex error")
+            }
+        } else {
+            print("Invalid RGB string, missing '#' as prefix", terminator: "")
+        }
+
+        self.init(red:red, green:green, blue:blue, alpha:alpha)
+    }
+
+    static var windowBackgroundColor: UIColor {
```

Comment:
static var/let are lazy initalized so it will be kept in memory only after being instantiated.
In an about car, I keep the palette of color in a static let, and use method/computed propertie to keep the naming.

```Swift
private struct Palette {
    static let whiteColor = UIColor(white: 255.0 / 255.0, alpha: 1.0)
    static let royalBlueColor = UIColor(red: 55.0 / 255.0, green: 109.0 / 255.0, blue: 237.0 / 255.0, alpha: 1.0)
}

extension UIColor {
    // MARK: Styles
    class func sectionHeaderBackgroundColor() -> UIColor {
        return Palette.royalBlueColor
    }
}
```
But it's no the point of this PR, keep it as it is, and we can discuss it outside of this PR.